src/script/conversation/ConversationRepository.ts

```diff
@@ -2548,11 +2534,7 @@ export class ConversationRepository {
       if (previouslyArchived) {
         // Add to check for un-archiving at the end of stream handling
         if (eventFromStream) {
-          return this.conversationsWithNewEvents.set(conversationEntity.id, conversationEntity);
-        }
-
-        if (eventFromWebSocket && conversationEntity.shouldUnarchive()) {
-          return this.unarchiveConversation(conversationEntity, false, 'event from WebSocket');
+          this.conversationsWithNewEvents.set(conversationEntity.id, conversationEntity);
```

Comment:
Do you know what is this `conversationWithNewEvents` map for?

Reply:
that is not totally clear to me. I checked all the occurrence of it, its used only in this file. we are creating a map and setting it here with conversationEntity.

Reply:
Please check but I think you can remove it now (since it's not used to unarchive convo)

Reply:
checked and removed